examples/pytorch/infograph/qm9_v2.py

```diff
@@ -0,0 +1,213 @@
+import numpy as np
+import os
+from tqdm import tqdm
+
+import torch as th
+
+import dgl
+from dgl.data.dgl_dataset import DGLDataset
+from dgl.data.utils import download, load_graphs, _get_dgl_url, extract_archive
+
+class QM9Dataset_v2(DGLDataset):
```

Comment:
DGL follows CamelCase naming convention for classes. "QM9Dataset_v2" -> "QM9DatasetV2".